README.md

```diff
@@ -426,11 +373,7 @@ const optimizely = createInstance({
 });
 ```
 

```

Comment:
Line 295: please [run a beta](https://docs.developers.optimizely.com/rollouts/docs/run-a-beta) to [run a beta](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/run-a-beta)